### PR TITLE
checkmark for side bar global parameters

### PR DESCRIPTION
**Pull request by vsrisuknimit**
What I did:
- Check mark for the global parameters on the side bar

What I would like to do:
-  ~~make the checkmark green instead. I don't know how to do css inside v-html.~
- ~~change GEN count to LAST instead of a number after the three main global parameters are reached.~~
- ~~not use hard code number of max ocean, max oxygen, etc. Is the only way to avoid is to pass more props?~~

image

### CODE REVIEW

**Comment by vsrisuknimit** [comment]
I was debugging. I set the max temp and max ocean to be -26 and 2 to test.

**Comment by nwai90** [comment]
@vsrisuknimit You can use the constants `MAX_OCEAN_TILES`, `MAX_OXYGEN_LEVEL` and `MAX_TEMPERATURE`. Same for Venus

**Comment by vsrisuknimit** [comment]
@nwai90 would it know inside vue in preferences.ts?

**Comment by vsrisuknimit** [comment]
Thanks! Using global constants now.

**Comment by vsrisuknimit** [comment]
- Added green check instead.
- Decide not to LAST of GEN, users can already see 3 check marks for the main parameters. That might be something to consider for solo games.

Good to go now.

File: src/components/Preferences.ts [hunk 1]
```diff
@@ -92,28 +92,28 @@ export const Preferences = Vue.component("preferences", {
         },
         getOceanCount: function(): string{
             if (this.oceans === MAX_OCEAN_TILES){
-                return `<img width="18" src="/assets/checkmark.png" alt="completed">`;
+                return `<img width="18" src="/assets/checkmark.png" style="filter: hue-rotate(280deg);" alt="completed">`;
```
Comment (alrusdi) [inline src/components/Preferences.ts:95]: Please move width and filter properties to css (less) file

**Review by alrusdi — CHANGES_REQUESTED** [review]

File: src/components/Preferences.ts [hunk 2]
```diff
@@ -92,28 +92,28 @@ export const Preferences = Vue.component("preferences", {
...
             } else {
                 return `${this.oceans}`;
             } 
         },
         getTemperatureCount: function(): string{
             if (this.temperature === MAX_TEMPERATURE){
-                return `<img width="18" src="/assets/checkmark.png" alt="completed">`;
+                return `<img width="18" src="/assets/checkmark.png" style="filter: hue-rotate(280deg);" alt="completed" style="">`;
```
Comment (alrusdi) [inline src/components/Preferences.ts:102]: Try to make **alt** property translatable.
1) Import $t function from directives/i18n
2) Use it like **:alt="$t('Completed!')"**

**Review by alrusdi — CHANGES_REQUESTED** [review]

File: src/components/Preferences.ts (hunk 1 above)
Comment (vsrisuknimit) [reply src/components/Preferences.ts]: Moved it to css

File: src/components/Preferences.ts (hunk 2 above)
Comment (vsrisuknimit) [reply src/components/Preferences.ts]: Added translatable alt text.

**Comment by vsrisuknimit** [comment]
Added new icon and moved the png file to assets/misc folder.

**Review by alrusdi — APPROVED** [review]

**Pull request merged**